test/unit/BibPage.test.js

```diff
@@ -147,9 +156,14 @@ describe('BibPage', () => {
             fromUrl: 'resultsurl.com',
             bibId: 'wrongbib',
           }}
-        />, { context });
-
-      expect(component.find('Link').length).to.equal(0);
+        />,
+        { context },
+      );
+
+      expect(component.find(BackToSearchResults)).to.have.lengthOf(1);
+      expect(
+        component.find(BackToSearchResults).first().render().find(Link).length,
+      ).to.equal(0);
```

Comment:
```suggestion
        component.find(BackToSearchResults).first().render().text().length,
      ).to.equal(0);
```

I think it would make more sense to look for any rendered text instead of the Link component.

Reply:
Sorry, I don't follow. Why is it better to check for the text instead of the link? Generally we'd want tests to be more specific.

Reply:
Well, for specificity purposes the text "Back To Search Results" would comply because the "Link" could be passed any text where as the "Text" must match the desired pupose.